### PR TITLE
Create focus-group-summaries-2019.md

### PR DESCRIPTION
**Pull request by GeorgiaHCA**
This PR creates a markdown document for the summaries for the 2019 focus groups, and contains the focus group held on the 18th. 

I'd appreciate comments on: 
the formatting
any typos or corrections which need to be made? 

I will add in another commit the other two summaries, and a contents page and link

### CODE REVIEW

**Comment by GeorgiaHCA** [comment]
All three summaries are now added, but some extra formatting is still needed: 

- [ ] Linking to all dates from the top
- [ ] Making sure headings are consistent
- [ ] Creating a contents page with links
- [ ] Creating a table for the summary
- [ ] Proof read

**Comment by GeorgiaHCA** [comment]
I've now reformatted and added internal links

File: community-recommendations/focus-group-summaries-2019.md [hunk 1]
```diff
@@ -0,0 +1,1511 @@
+# Focus Group Summaries
+
+This file contains summaries and quotations from focus group sessions concerning the Autistica/Turing citizen science project, which was held with members of Autistica's Insight group, Fujitsu, The Turing, and Autistica on the 18th and 24th September 2019, and 4th October 2019. Quotations have been shortened or summarised, labelled according to the speaker, and all depersonalising information has been removed.
```
Comment (KirstieJane) [inline community-recommendations/focus-group-summaries-2019.md:3]: Please put different sentences on different lines.

File: community-recommendations/focus-group-summaries-2019.md [hunk 2]
```diff
@@ -0,0 +1,1511 @@
...
+5. [Representing others](https://github.com/alan-turing-institute/AutisticaCitizenScience/blob/GeorgiaHCA-Summary1/community-recommendations/focus-group-summaries-2019.md#5-representing-others)
+6. [Data management](https://github.com/alan-turing-institute/AutisticaCitizenScience/blob/GeorgiaHCA-Summary1/community-recommendations/focus-group-summaries-2019.md#6-data-management)
+7. [Contribution channels](https://github.com/alan-turing-institute/AutisticaCitizenScience/blob/GeorgiaHCA-Summary1/community-recommendations/focus-group-summaries-2019.md#7-contribution-channels)
+8. [Ways of working](https://github.com/alan-turing-institute/AutisticaCitizenScience/blob/GeorgiaHCA-Summary1/community-recommendations/focus-group-summaries-2019.md#8-ways-of-working)
+	
+* [24th September 2019](https://github.com/alan-turing-institute/AutisticaCitizenScience/blob/GeorgiaHCA-Summary1/community-recommendations/focus-group-summaries-2019.md#citizen-science-focus-group-24092019)
+
+1. [Value of project and motivations to take part](https://github.com/alan-turing-institute/AutisticaCitizenScience/blob/GeorgiaHCA-Summary1/community-recommendations/focus-group-summaries-2019.md#8-ways-of-working)
```
Comment (KirstieJane) [inline community-recommendations/focus-group-summaries-2019.md:24]: For the links please use relative links so you aren't stuck to a specific branch.

**Comment by GeorgiaHCA** [comment]
Closing this PR as md summaries have been merged and are available in the community-recommendations folder

**Pull request closed without merging**